### ISSUE TITLE
"Trakt List" import unable to get default Trakt Watchlist's

### ISSUE THREAD

**Issue by jmengit** [Radarr/Radarr#5542]
**Is your feature request related to a problem? Please describe.**
When adding a list of style "Trakt List" and defining the list name as Trakt's default user watchlist (named simply "Watchlist") it fails to fetch the Trakt users list. 

**Describe the solution you'd like**
As it stands, the only way to get Trakt's default "Watchlist" for a given user is to use the list type "Trakt User" in which Radarr authenticates with Trakt and then pulls the authenticated users list "Watchlist". This limits you to following only ONE users "Watchlist" as only one user can be authenticated with Radarr.

**Describe alternatives you've considered**
You should be able to follow multiple Trakt users default "Watchlist" by adding them as a "Trakt List" type list and entering a trakt username and "List Name" of "Watchlist". This functionality was present under Radarr v2. 

**Additional context**
I think the problem is fairly straightforward. Trakt stores all user lists at "https://trakt.tv/users/username/list/listname" EXCEPT for the default watchlist which is accessed at "https://trakt.tv/users/username/watchlist". So, as it stands, if you use the "List Name" of "Watchlist" under a list type of "Trakt List" it will look to fetch the list at "https://trakt.tv/users/username/list/Watchlist" which is incorrect. Implementing logic to look at "https://trakt.tv/users/username/watchlist" IF the List Name is "Watchlist" seems like the most straightforward solution.

**Comment by vorte** [comment]
I am having the same issue. The ability to add multiple user's watchlists was available in v2 but is missing in v3

**Comment by jmengit** [comment]
> 
> 
> I am having the same issue. The ability to add multiple user's watchlists was available in v2 but is missing in v3

Yep. Totally agree. This was easily accomplished in v2. Surprised this feature request isnt getting more traction. Perhaps should have added it as a bug.

**Comment by austinwbest** [comment]
Bug doesn't get anymore weight than a feature when it comes to discussion and resolving. It is a matter of who happens to see what issue & when.

**Comment by jmengit** [comment]
> 
> 
> Bug doesn't get anymore weight than a feature when it comes to discussion and resolving. It is a matter of who happens to see what issue & when.

Great. Wasnt sure if there was hierarchy. Thanks for the input.

**Comment by aguillem** [comment]
Same bug for me, a workaround is to create a public list and connect to it instead of watchlist.

**Comment by briankelley** [comment]
> 
> 
> Same bug for me, a workaround is to create a public list and connect to it instead of watchlist.

Literally installed Radarr for the first time ever yesterday and the comment by @aguillem is the only way I was able to add a trakt list to it. I probably spent 3 hours trying to get this working. Thanks @aguillem

This issue is not "stale". It still exists.

**Comment by Arthur1967** [comment]
This may be a stupid question but I just downloaded the latest version of it now what do I use it with her on sorry for being so green about this but I'd like to try it thank you￼

**Comment by jmengit** [comment]
Long story short you cannot monitor multiple Trakt users default watchlists. You can monitor one Trakt users watchlist by choosing the "Trakt User" option in radarr's lists (this requires login). For the other trakt users you will need them to create a public list and add movies there and then use the "Trakt List" option in radarr.

**Comment by briankelley** [comment]
@Arthur1967 This board is for bug reports. Go to https://www.reddit.com/r/radarr for community help or read the wiki https://wiki.servarr.com/radarr

**Comment by Arthur1967** [comment]
OK will do thank you

**Comment by briankelley** [comment]
> 
> 
> Long story short you cannot monitor multiple Trakt users default watchlists. You can monitor one Trakt users watchlist by choosing the "Trakt User" option in radarr's lists (this requires login). For the other trakt users you will need them to create a public list and add movies there and then use the "Trakt List" option in radarr.

Yep. The post was about pulling the default list though. Making a new public list is what worked. Thanks for putting the issue up @jmengit

**Comment by Fazzani** [comment]
The username must respect the following format
toto.tata@gmail.com ==>  toto-tata-gmail-com
